src/main/java/com/skankhunt220/api/controller/UserController.java

```diff
@@ -16,31 +20,28 @@
 public class UserController {
     private final UserService userService;
 
-    @GetMapping("/allUsers")
-    public List<UserDto> getAllUsers() throws IOException {
-        List<UserDto> listDto = new ArrayList<>();
-        userService.readAll().forEach(user -> listDto.add(UserTransformer.transformIntoUserDto(user)));
-
-        return listDto;
+    @GetMapping(value = "/{spreadsheetId}", params = {"page", "size"})
+    public Page<UserDto> readAll(@RequestParam("page") int page, @RequestParam("size") int size, @PathVariable("spreadsheetId") String spreadsheetId) throws IOException {
```

Comment:
Don`t use primitives for this parameters, use Object, primitives has a default value. Аlso use a default value on the page and size so that it can be omitted in the link